### DISCUSSION TITLE
Vercel deployment is stuck as Queued since last 24 hrs

### DISCUSSION THREAD

**Discussion by SachinKotnala** [vercel/community#3269] [Bug Report]
### Page to Investigate

insight-mine-3p8dg8k1v-sachinkotnala.vercel.app

### Steps to Reproduce

I thought service might be down for some time but all new deployments are also getting Queued

**Comment by SachinKotnala** [comment]
It's fixed on it's own.

**Comment by swarnava** [comment]
There was an issue on our end and we processed the queue. It should be working now.
